apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -311,7 +311,7 @@ const ExpandedCurriculumCatalogCard = ({
                   src={recommendedSimilarCurriculum.image || defaultImageSrc}
                   alt={recommendedSimilarCurriculum.display_name}
                 />
-                <Button
+                <DeprecatedButton
```

Comment:
I think you can use the [Link component](https://code-dot-org.github.io/cdo-styleguide/?path=/docs/designsystem-link--docs) in place of this!

Reply:
It looks like the Link component only supports anchor tags, right? Semantically, shouldn't this be a button?

Reply:
Are these the links here?

<img width="223" alt="Screenshot 2024-07-24 at 9 04 57 AM" src="https://github.com/user-attachments/assets/45b83503-aa30-4683-8ad4-25a5af58bacf">

Reply:
Yep!

Reply:
ooo I guess because it's going to a new location on the same page, it might be okay as an anchor tag 🤔

Reply:
That's what I was thinking originally, it's def a trickier use case! Let's go with the anchor tag since that's how it's behaving to a user, even if a button technically could work too.